INFO     root:test_15_servermonitor.py:28 ==========开始执行用例：servermonitor_serverlist_01==========
INFO     root:runcase.py:30 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:47 重组后的用例信息：{'name': '查询服务器列表', 'description': '查询服务器列表', 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '系统管理', 'feature': '运维管理-服务监控', 'story': '查询服务器列表接口', 'base_url': 'https://192.168.148.174:31000', 'request': {'method': 'GET', 'address': '/api/auth/v1/monitor/servers', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjY1OTBhNTE1LTA0Y2EtNDA5ZS1hNzBmLWMxZDk2MDY1MTU5MiIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI2NTkwYTUxNS0wNGNhLTQwOWUtYTcwZi1jMWQ5NjA2NTE1OTIiLCJleHAiOjE2ODgwMzE2MjMsImlhdCI6MTY4Nzg1MTYyM30.ZQ97PBbwO0FxfTBcZAH7cNMO0z8M0fncEgMdMVNutTK5fS6qwMFhZur757v9anwBP_z2errE-UoAcB88J2n1ag', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/sysManage/systemOperation/operationMonitoring/index'}}, 'caseid': 'servermonitor_serverlist_01'}
INFO     root:runcase.py:83 请求地址：https://192.168.148.174:31000/api/auth/v1/monitor/servers
INFO     root:runcase.py:84 请求方法：GET
INFO     root:runcase.py:85 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjY1OTBhNTE1LTA0Y2EtNDA5ZS1hNzBmLWMxZDk2MDY1MTU5MiIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI2NTkwYTUxNS0wNGNhLTQwOWUtYTcwZi1jMWQ5NjA2NTE1OTIiLCJleHAiOjE2ODgwMzE2MjMsImlhdCI6MTY4Nzg1MTYyM30.ZQ97PBbwO0FxfTBcZAH7cNMO0z8M0fncEgMdMVNutTK5fS6qwMFhZur757v9anwBP_z2errE-UoAcB88J2n1ag', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/sysManage/systemOperation/operationMonitoring/index'}
DEBUG    urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
DEBUG    urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "GET /api/auth/v1/monitor/servers HTTP/1.1" 200 268
INFO     root:runcase.py:168 请求接口结果： {'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 21334, 'list': [{'id': 52611, 'modifyTime': '2023-06-27T07:44:16.000+00:00', 'endpointId': 1, 'serverIp': '192.168.148.174', 'cpuUsed': '23.653198648328715', 'memoryUsed': '48.763360187580965', 'diskUsed': '3.8456888128969227'}]}}
INFO     root:checkresult.py:113 code断言通过, 期望状态码:200, 实际状态码:200
INFO     root:checkresult.py:48 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 21334, 'list': [{'id': 52611, 'modifyTime': '2023-06-27T07:44:16.000+00:00', 'endpointId': 1, 'serverIp': '192.168.148.174', 'cpuUsed': '23.653198648328715', 'memoryUsed': '48.763360187580965', 'diskUsed': '3.8456888128969227'}]}}
INFO     root:runcase.py:63 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-